README.md

```diff
@@ -19,12 +19,24 @@ You can do it on fedora with next comand:
 sudo dnf install libftdi-devel libusb-devel
 ```
 
+For Ubuntu OS need to install packages by running the command:
+
+```
+sudo apt install -y libftdi-dev libusb-dev libusb-1.0-0-dev libftdi1-dev
+```
+
 After that simply run make:
 
 ```
 make
 ```
 
+Example of running the utility on host where connected only one EvB:
+
+```
+sudo ./rff-reset --only-one
```

Comment:
With right privileges set there's no need to use `sudo`, so let's not add confusion here and remove `sudo`. Remember not every user (especially in big corporate environments) may have rights to execute `sudo` ;)

Reply:
"sudo" removed.